### DISCUSSION TITLE
How to use `rewrites` on runtime

### DISCUSSION THREAD

**Discussion by nstfkc** [vercel/next.js#22872] [Help]
Hello there, 

Is there any way to change `rewrites` array after build. 

Reason:
We are currently running our app on multiple markets under different domains. And we have different urls for the same page e.g.
// France
`/liste-d-envies` -> `pages/wishlist.tsx`
// Germany
`/merkzettel` -> `pages/wishlist.tsx`

So we want to be able to change rewrites based on the deployment. To prevent having both rewrites enabled in both markets.

Cheers!

**Comment by JulienKode** [comment]
I'm interesting to see the answer
